lib/bundlex/toolchain/common/unix/os_deps.ex

```diff
@@ -131,18 +131,24 @@ defmodule Bundlex.Toolchain.Common.Unix.OSDeps do
     end
   end
 
-  defp get_precompiled_libs_flags(dep_path, dep_dir_name, lib_names, native) do
+  defp get_precompiled_libs_flags(dep_path, logical_dep_dir_name, lib_names, native) do
     lib_path = Path.join(dep_path, "lib")
-    output_path = Bundlex.Toolchain.output_path(native.app, native.interface)
-    create_relative_symlink(lib_path, output_path, dep_dir_name)
+    logical_output_path = Bundlex.Toolchain.output_path(native.app, native.interface)
+    create_relative_symlink(lib_path, logical_output_path, logical_dep_dir_name)
+
+    {physical_output_path, 0} = System.shell("realpath #{logical_output_path}")
```

Comment:
let's add a comment on why we do this and create two symlinks

Reply:
done